cvat-core/src/session.js

```diff
@@ -386,10 +386,10 @@
                 * @throws {module:API.cvat.exceptions.ArgumentError}
             */
             /**
-                * Get the first frame of a task for preview
```

Comment:
Do not delete documentation for the method which is necessary in current develop.